.circleci/config.yml

```diff
@@ -7,7 +7,7 @@
 defaults: &defaults
   working_directory: ~/italia-app
 
-node_cache_key: &node_cache_key v1.10-dependencies-{{ checksum "yarn.lock" }}
+node_cache_key: &node_cache_key v1.11-dependencies-{{ checksum "yarn.lock" }}
```

Comment:
I merged [this PR](https://github.com/teamdigitale/io-app/pull/1514) to solve the problem 
You can revert this change